frontend/src/scenes/pipeline/hogfunctions/list/hogFunctionTemplateListLogic.tsx

```diff
@@ -92,15 +92,17 @@ export const hogFunctionTemplateListLogic = kea<hogFunctionTemplateListLogicType
             },
         ],
     })),
-    loaders(({ props }) => ({
+    loaders(({ props, values }) => ({
         templates: [
             [] as HogFunctionTemplateType[],
             {
                 loadHogFunctionTemplates: async () => {
+                    const dbTemplates = !!values.featureFlags?.getTemplatesFromDB
```

Comment:
Try and look for existing examples as a reference - we have standard here.

`!values.featureFlags[FEATURE_FLAGS.DISCUSSIONS]` - we use the `FEATURE_FLAGS` constant which contains the list of flags rather than hard coded values everyhwere _and_ we use a specific casing.